src/diet.ts

```diff
@@ -129,12 +144,35 @@ function useIfUnused(item: Item, prop: string | boolean, maxPrice: number) {
   }
 }
 
-const valuePerSpleen = (item: Item) => (adventureGain(item) * MPA - mallPrice(item)) / item.spleen;
+function estimatedTurnsWithOrgans() {
+  const fullnessAvailable =
+    fullnessLimit() +
+    (have($item`distention pill`) && !get("_distentionPillUsed") ? 1 : 0) +
+    (mallPrice($item`spice melange`) < 50 * MPA ? 3 : 0);
+  const inebrietyAvailable =
+    inebrietyLimit() +
+    (have($item`synthetic dog hair pill`) && !get("_syntheticDogHairPillUsed") ? 1 : 0) +
+    (mallPrice($item`spice melange`) < 50 * MPA ? 3 : 0);
+  const spleenAvailable = spleenLimit() + (3 - get("currentMojoFilters"));
+  return (
+    estimatedTurns() +
+    8 * Math.max(0, fullnessAvailable - myFullness()) +
+    8 * Math.max(0, inebrietyAvailable - myInebriety()) +
+    2 * Math.max(0, spleenAvailable - mySpleenUse()) +
+    (have($familiar`Pocket Professor`) && !get("_thesisDelivered") ? 11 : 0)
```

Comment:
thumb ring multiplier for organ & prof turns--should we not just add thesis turns to the estimatedTurns() function?

Reply:
yep